packages/appstore/src/constants/wallets-tour-steps-config.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
```

Comment:
@aizad-deriv better to add this to the `tour-step-config` instead of making new one

Reply:
@aizad-deriv I also have another question, do we use `tour-step-config` anywhere in the code? if not you can remove it